lib/ddtrace/transport/http/response.rb

```diff
@@ -11,13 +10,49 @@ module HTTP
       # Used by endpoints to wrap responses from adapters with
       # fields or behavior that's specific to that endpoint.
       module Response
-        extend ::Forwardable
-
         def initialize(http_response)
           @http_response = http_response
         end
 
-        def_delegators :@http_response, *Datadog::Transport::Response.instance_methods
+        # (see Datadog::Transport::Response#payload)
+        def payload
+          @http_response.payload
+        end
+
+        # (see Datadog::Transport::Response#internal_error?)
+        def internal_error?
+          @http_response.internal_error?
+        end
+
+        # (see Datadog::Transport::Response#unsupported?)
+        def unsupported?
+          @http_response.unsupported?
+        end
+
+        # (see Datadog::Transport::Response#ok?)
+        def ok?
+          @http_response.ok?
+        end
+
+        # (see Datadog::Transport::Response#not_found?)
+        def not_found?
+          @http_response.not_found?
+        end
+
+        # (see Datadog::Transport::Response#client_error?)
+        def client_error?
+          @http_response.client_error?
+        end
+
+        # (see Datadog::Transport::Response#inspect)
+        def inspect
+          @http_response.inspect
+        end
```

Comment:
I know this was already like this, but I suspect this was not intended. This `inspect` makes this module and the class that hosts it disappear when one's debugging, which can be quite confusing. If we remove this and use the class's own inspect, we'll get much better output.

```suggestion
```

Reply:
Very good point!